source/guide_offen.rst

```diff
@@ -0,0 +1,326 @@
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ offen.yourdomain.org
+ [isabell@stardust ~]$
+
+.. note:: This subdomain setup is required so that Offen can set cookies from
+    the same domain that your target site is running on. I.e. if your site is
+    running on `www.yourdomain.org`, Offen is ideally running on
+    `offen.yourdomain.org`. You could use the `isabell.uber.space` domain if you
+    want to, but that will make Offen issue 3rd party cookies, which are subject
+    to blocking in many scenarios. A domain can be added at a later point in
+    time though.
+
+.. include:: includes/my-print-defaults.rst
+
+Installation
+============
+
+.. _Download:
+
+Download the binary distribution
+------
+
+Offen is distributed as a single binary file. To kick off your install, download
+the latest release:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/tmp/offen-download
+ [isabell@stardust ~]$ curl -sSL https://get.offen.dev | tar -xvz -C ~/tmp/offen-download
+ LICENSE
+ NOTICE
+ README.md
+ checksums.txt
+ offen-darwin-10.6-amd64
+ offen-darwin-10.6-amd64.asc
+ offen-linux-amd64
+ offen-linux-amd64.asc
+ offen-windows-4.0-amd64.exe
+ offen-windows-4.0-amd64.exe.asc
+ [isabell@stardust ~]$
+
+.. note:: The distribution tarball also contains non-Linux binaries which is why
+    you also see those `darwin` and `windows` files. We will delete them later.
+
+Next, we should check if the binary matches the signature - i.e. it has not been
+altered by 3rd parties - using GPG (this step is optional, but `highly
+recommended`):
+
+::
+
+ [isabell@stardust ~]$ curl -sSL https://keybase.io/hioffen/pgp_keys.asc | gpg --import
+ gpg: key C90B8DA1: public key "Offen (Signing Binaries) <hioffen@posteo.de>" imported
+ gpg: Total number processed: 1
+ gpg:               imported: 1  (RSA: 1)
+ gpg: no ultimately trusted keys found
+ [isabell@stardust ~]$ gpg --verify ~/tmp/offen-download/offen-linux-amd64.asc ~/tmp/offen-download/offen-linux-amd64
+ gpg: Signature made Mo 13 Jul 2020 18:05:28 CEST using RSA key ID C90B8DA1
+ gpg: Good signature from "Offen (Signing Binaries) <hioffen@posteo.de>"
+ gpg: WARNING: This key is not certified with a trusted signature!
+ gpg:          There is no indication that the signature belongs to the owner.
+ Primary key fingerprint: F20D 4074 068C 636D 58B5  3F46 FD60 FBED C90B 8DA1
+ [isabell@stardust ~]$
+
+.. note:: GPG will print warnings about the signing key not being certified
+    as your Uberspace's keyring is likely empty and does therefore not know
+    about anyone who has signed Offen's key. `This is expected behavior`. You
+    can check the Offen Keybase_ profile for further proof that this key is
+    the correct one.
+
+Install the command
+-----
+
+Next, you can install the Linux binary on your Uberspace:
+
+::
+
+ [isabell@stardust ~]$ cp ~/tmp/offen-download/offen-linux-amd64 ~/bin/offen
+ [isabell@stardust ~]$ which offen
+ ~/bin/offen
+ [isabell@stardust ~]$
+
+Cleaning up
+-----
```

Comment:
Please make sure the length of the `----` likes matches the length of `Cleaning up` to get rid of build warnings. Same for all following headings.